tada_a_bigoh.py

```diff
@@ -281,11 +281,14 @@ def main():
         table, result = tada(arg)
         resultstables.append(table)
         tada_results.update(result)
+        contrast_flag = arg.contrast
     # display results
     for table in resultstables:
         results.display_resultstable(table, tada_arg_list[0].md)
     if len(resultstables) > 1:
         results.compare(*results.greatest_common_size(tada_results))
+    if contrast_flag is True:
```

Comment:
I think this if condition should go nested in the `len(resultstables) > 1` otherwise there would be an error if only one function is being given

Reply:
I think you are right, fixed